lib/actions.ex

```diff
@@ -213,7 +212,7 @@ defmodule EctoShorts.Actions do
 
       iex> {:ok, schema} = EctoSchemas.Actions.find_and_update(EctoSchemas.Accounts.User, %{email: "some_email"}, %{name: "great name}, repo: MyApp.MyRepoModule.Repo, replica: MyApp.MyRepoModule.Repo.replica())
   """
-  @spec find_and_update(query(), map, map, opts) :: {:ok, Ecto.Schema.t()} | {:error, Ecto.Changeset.t()}
+  @spec find_and_update(query(), map, map, opts) :: {:ok, Ecto.Schema.t()} | {:error, any()}
```

Comment:
This and all the other ones where I made this change can also return `{:error, ErrorMessage.t()} ` but since the latter can be swapped for anything, it's {:error, any()}